src/world/area_kmr/kmr_00/kmr_00_3_revival.c

```diff
@@ -103,16 +103,30 @@ API_CALLABLE(N(func_802405F0_8AC580)) {
         FUNC_STATE_3        = 3
     };
 
+#if VERSION_PAL
+#define tempVar script->varTable[11]
```

Comment:
Going forward, please ALL_CAPS for these to distinguish from local variables.

Reply:
I went looking through the codebase for precedent, and I was wondering if there's a preference for including the `script->` part or not. In `world/partner/goombario.c`, there's `#define USE_STATE functionTemp[0]`, used as `script->USE_STATE++;`

Reply:
We've had discussions about it, but not widely implemented them. Leaving off the script part is preferred.